docs/model.md

```diff
@@ -5,45 +5,51 @@ The formal [model specification](https://github.com/open-traffic-generator/model
 
 * **Test Ports** with Layer 1&2 capabilities, including:
 	- LLDP, LAG, LACP
-* Emulated **OTG Devices** with Layer 2&3 features:
+* **Emulated Devices** with Layer 2&3 features:
 	- IPv4, IPv6 interfaces
 	- ARP, IPv6 ND
 	- BGP, IS-IS routing protocols
 * **Traffic Flows** 
-    - associated with either Test Ports, or OTG Devices
+    - associated with either Test Ports, or Emulated Devices
 	- expressing L2-4 properties like Ethernet, IPv4/IPv6, TCP/UDP
 	- stateless or stateful capabilities for transport protocols
 	- with implementation-specific application payload
 * Run-time **Metrics** and traffic **Capture** capabilities
 
 ## Hierarchy
 
-The hierarchy of objects the OTG Model is visualized below.
-<!-- TODO replace with an image from the images subfolder -->
-![OTG Hierarchy](https://raw.githubusercontent.com/open-traffic-generator/models/docs/docs/overview.drawio.svg)
+The hierarchy of objects in the OTG Model is visualized below.
+
+![OTG Hierarchy](images/otg-model-hierarchy.svg)
 <p style="text-align: center;"><sub>Fig. 1. Hierarchy of the OTG objects</sub></p>
 
 ##Raw Traffic Flows
  
-In the most simple cases, the OTG Model describes **Raw Traffic Flows**: stateless streams of packets to be transmitted from one Test Port, and expected to be received on another Test Port.
+In the most simple case, the OTG Model describes **Raw Traffic Flows**: stateless streams of packets to be transmitted from one Test Port, and typically, but not always, expected to be received on another Test Port. A configuration when a flow is not expected to be received, is called *one-arm*. Otherwise, it is a *two-arm* configuration.
+
+![Raw Traffic Flow](images/otg-model-raw-flows.svg)
+<p style="text-align: center;"><sub>Fig. 2. Two-arm configuration with a Raw Traffic Flow</sub></p>
+
+Since Traffic Flows are unidirectional and stateless, a bidirectional communication can be expressed by two Traffic Flows running on opposite directions.
+
+![Opposite Raw Traffic Flows](images/otg-model-raw-flows-bi.svg)
+<p style="text-align: center;"><sub>Fig. 3. Configuration with two Raw Traffic Flow representing a bidirectional comminication</sub></p>
 
-![Raw Traffic Flows](images/otg-model-raw-flows.svg)
-<p style="text-align: center;"><sub>Fig. 2. Configuration with Raw Traffic Flows</sub></p>
 
 ##Devices and Flows
  
-Traffic Flows can also be associated with emulated **OTG Devices** to form 1:1 or mesh communications between them. Such approach allows to use the same Flow definition to originate traffic from multiple ports, as well as Link Aggregation Groups (LAGs).
+Traffic Flows can also be associated with **Emulated Devices** to form 1:1 or mesh communications between them. Such approach allows to use of the same Flow definition to originate traffic from multiple ports, as well as Link Aggregation Groups (LAGs).
```

Comment:
Such approach allows to use of the same  --> Such an approach allows the use of the same